src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { IconChevronDown } from '../Icons';
+
+import classNames from 'classnames';
+import styles from './Accordion.module.scss';
+
+const cx = classNames.bind(styles);
+
+export interface AccordionItem {
+  title: string;
+  text: string;
+}
+export interface AccordionProps {
+  items: AccordionItem[];
+  contrast?: 'high' | 'low';
+}
+
+export const Accordion = ({ items, contrast = 'high' }: AccordionProps) => {
+  const [focusState, setFocusState] = useState(false);
+  return (
+    <div className={styles.Accordion} data-contrast={contrast}>
+      <RadixAccordion.Root className={styles.AccordionRoot} type="single" collapsible>
+        {items.map((itm, i) => (
+          <RadixAccordion.Item
+            className={styles.AccordionItem}
+            value={`item-${i}`}
+            key={i}
+            data-focus-on-key-press={focusState}
+          >
+            <RadixAccordion.Header className={styles.AccordionHeader}>
+              <RadixAccordion.Trigger
+                className={styles.AccordionTrigger}
+                onMouseDown={() => setFocusState(false)}
+                onKeyDown={e => setFocusState(true)}
```

Comment:
Remove the `e` if not using it, i.e. `() => setFocusState(true)`

Reply:
removed.